### ISSUE TITLE
Parsing error when using materialx.emoji.twemoji

### ISSUE THREAD

**Issue by apinter** [openSUSE/openSUSE-docs-revamped#48]
Every time I run `mkdocs build` I ran into a parsing error for `emoji_index: !!python/name:materialx.emoji.twemoji` and `emoji_generator: !!python/name:materialx.emoji.to_svg`.

```
yaml.constructor.ConstructorError: could not determine a constructor for the tag 'tag:yaml.org,2002:python/name:materialx.emoji.twemoji'
  in "/home/apinter/Project/git/openSUSE-docs-revamped/project/mkdocs.yml", line 51, column 22
```

Digging around a bit it seems that the issue is that mkdocs loses the python name tag values and/or unable to handle `!!`.

https://github.com/readthedocs/readthedocs.org/issues/6889
https://github.com/readthedocs/readthedocs.org/issues/6889#issuecomment-754060041

So there is that. Now the question: do we even need emojis in our docs?

**Comment by Nycticorax** [comment]
> Every time I run `mkdocs build` I ran into a parsing error for `emoji_index: !!python/name:materialx.emoji.twemoji` and `emoji_generator: !!python/name:materialx.emoji.to_svg`.
> 
> ```
> yaml.constructor.ConstructorError: could not determine a constructor for the tag 'tag:yaml.org,2002:python/name:materialx.emoji.twemoji'
>   in "/home/apinter/Project/git/openSUSE-docs-revamped/project/mkdocs.yml", line 51, column 22
> ```
> 
> Digging around a bit it seems that the issue is that mkdocs loses the python name tag values and/or unable to handle `!!`.
> 
> [readthedocs/readthedocs.org#6889](https://github.com/readthedocs/readthedocs.org/issues/6889)
> [readthedocs/readthedocs.org#6889 (comment)](https://github.com/readthedocs/readthedocs.org/issues/6889#issuecomment-754060041)
> 
> So there is that. Now the question: do we even need emojis in our docs?

Without emojis we are erecting an ivory tower under the docs, with no way for the 2021 gimmicks to make its way to the top. It's both irresponsible and elitist.
XD just kidding. I don't think we need emojis. The culprit being probably `pymdownx.emoji`, can someone test building with this markdown extension removed?

**Comment by apinter** [comment]
I already tested without emojis and it works fine. Not sure why is it there in the first place tbh. Anyhow, looking at the linked stuff this is considered an "unsafe" way of loading it by pyyaml: https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation 
And this is the reason why the potential solution is not merged yet.

Either way I'm fairly certain that we can live without emojis.

**Comment by Nycticorax** [comment]
Closing thanks to your fix from #49
